meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source /usr/sbin/resin-vars
+
+SUPERVISOR_IMAGE_ID=$(docker inspect --format='{{.Id}}' $SUPERVISOR_IMAGE)
+SUPERVISOR_CONTAINER_IMAGE_ID=$(docker inspect --format='{{.Image}}' resin_supervisor)
+
+runSupervisor() {
+    docker rm --force resin_supervisor || true
```

Comment:
The container might not be there at all, that's why (and it's how the service used to work, as it had a `-` in the ExecStartPre)